apps/graphql/src/apps/itinerary/helpers/__tests__/Itineraries.test.js

```diff
@@ -171,3 +172,10 @@ describe('differenceInMinutes', () => {
     expect(differenceInMinutes(null, toISO)).toBeNull();
   });
 });
+
+describe('mapToOpaqueID', () => {
+  it('returns proper formatted ID', () => {
+    const ids = ['STN', 'MGL'];
+    expect(mapToOpaqueID(ids)).toEqual(['I3', '0b']);
```

Comment:
Function `fromGlobalId` accepts opaque IDs and returns unmasked IDs. Therefore your function `mapToOpaqueID` is actually doing the opposite and this test is not correct. This violation will be fixed here: https://gitlab.skypicker.com/incubator/universe/merge_requests/1390 (in version 0.8)

Reply:
Thanks @mrtnzlml !! Still getting a bit confused about the different ID types.. 👍

Reply:
Basically, opaque ID (output from `@kiwicom/graphql-global-id`) represents ID from GraphQL specification (https://graphql.github.io/graphql-spec/draft/#sec-ID). The other IDs are just some strings floating through your application which I don't know much about. :)